helix-rest/src/main/java/org/apache/helix/rest/metadatastore/ZkMetadataStoreDirectory.java

```diff
@@ -179,10 +179,10 @@ public void refreshRoutingData(String namespace) {
     }
 
     if (_routingDataMap != null) {
-      MetadataStoreRoutingData newRoutingData =
-          new TrieRoutingData(new TrieRoutingData.TrieNode(null, null, false, null));
+//      MetadataStoreRoutingData newRoutingData =
+//          new TrieRoutingData(new TrieRoutingData.TrieNode(null, null, false, null));
       // TODO call constructRoutingData() here.
-      _routingDataMap.put(namespace, newRoutingData);
+//      _routingDataMap.put(namespace, newRoutingData);
```

Comment:
With the complete implementation of TrieRoutingData, you should be able to implement the logic here, correct?